maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,200 @@
+from ..logger import Generation, Logger, Trace
+from ..openai.utils import OpenAIUtils
+
+try:
+    from portkey_ai import AsyncPortkey, Portkey  # type: ignore
+except ImportError:
+    raise ImportError(
+        (
+            "The 'portkey-ai' package is required for Portkey integration. "
+            "Install it with `pip install portkey-ai` or `uv add portkey-ai`."
+        )
+    ) from exec
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)